### PR TITLE
[AC-9395] remove CAM V feature flagging

### PR DESCRIPTION
**Pull request by jonkiparsky**
https://masschallenge.atlassian.net/browse/AC-9395

All references to the CAM V feature flag (`activate_unified_profile_views') should be removed

### CODE REVIEW

File: accelerator_abstract/models/base_core_profile.py [hunk 1]
```diff
@@ -647,10 +638,7 @@ def participation(self):
 
     @property
     def user_default_page(self):
-        if flag_smith_has_feature('activate_unified_profile_views'):
-            url_map = LANDING_PAGE_MAP
-        else:
-            url_map = OLD_LANDING_PAGE_MAP
+        url_map = LANDING_PAGE_MAP
```
Comment (AmosWels) [inline accelerator_abstract/models/base_core_profile.py:641]: seems we don't need the `url_map` now

**Review by AmosWels — APPROVED** [review]
Changes look good, all `activate_unified_profile_views` instances have been removed

**Comment by iankigen** [comment]
potential migration conflict. current migration 0105

**Comment by iankigen** [comment]
it seems we have some unneeded commits

**Pull request merged**